content/en/continuous_integration/setup_tests/junit_upload.md

```diff
@@ -10,87 +10,89 @@ further_reading:
       text: "Troubleshooting CI"
 ---
 
-## Compatibility
-
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+JUnit test report files are XML files that contain test execution information, such as test and suite names, pass/fail status, duration, and sometimes error logs. Although it was introduced by the [JUnit][4] testing framework, lots of other popular frameworks are able to output results using this format.
+
+As an alternative to instrumenting your tests natively using Datadog tracers, which is the recommended option as it provides the most comprehensive test results, you can also upload JUnit XML test reports.
+
+Test results imported via JUnit XML reports will appear alongside tests reported via tracers. However, there are some limitations when using this method, such as the lack of distributed traces on integration tests or structured stacktraces. For this reason, only use this method if there is no native support for the language or testing framework being used.
 
 ## Installing the Datadog CI CLI
 
-Install the `@datadog/datadog-ci` CLI:
+Install the [`datadog-ci`][2] CLI globally using `npm`:
 
 {{< code-block lang="bash" >}}
-yarn global add @datadog/datadog-ci
+npm install -g @datadog/datadog-ci
 {{< /code-block >}}
 
 ## Uploading test reports
 
-To upload your JUnit XML test reports to Datadog:
+To upload your JUnit XML test reports to Datadog, run the following command, specifying the name of the service or library that was tested using the `--service` parameter, and one or more file paths to either the XML report files directly or directories containing them:
 
 {{< code-block lang="bash" >}}
-datadog-ci junit upload [--service] [--tags] [--max-concurrency] [--dry-run] <paths>
+datadog-ci junit upload --service <service_name> <path> [<path> ...]
 {{< /code-block >}}
 
-For example:
+You will need to specify a valid [Datadog API key][3] in the `DATADOG_API_KEY` environment variable, and the environment where tests were run (e.g. `local` when uploading results from a developer workstation, or `ci` when uploading them from a CI provider) in the `DD_ENV` environment variable. For example:
 
 {{< code-block lang="bash" >}}
-datadog-ci junit upload --service my-service \
-  --tags key1:value1 --tags key2:value2 \
-  unit-tests/junit-reports acceptance-tests/junit-reports e2e-tests/single-report.xml
+export DATADOG_API_KEY=a5f...ea170229
+export DD_ENV=ci
+datadog-ci junit upload --service my-api-service unit-tests/junit-reports e2e-tests/single-report.xml
 {{< /code-block >}}
 
+## Additional configuration settings
+
+This is the full list of options available when using the `datadog-ci junit upload` command:
+
 `--service`
-: The name of the service you're uploading JUnit XML reports for.<br>
-**Default**: the `DD_SERVICE` environment variable value.
+: Name of the service or library under test.<br/>
```

Comment:
can we consider `--service` "additional"? It is mandatory and explained earlier

Reply:
Maybe "additional" is not the best title for the section, but I think it's good to include service and environment in this list even if they are explained earlier, just for completeness. Maybe we can use "Advanced configuration" or something like that.